src/signature_pad.ts

```diff
@@ -46,6 +46,8 @@ export interface Options extends Partial<PointGroupOptions> {
 
 export interface PointGroup extends PointGroupOptions {
   points: BasicPoint[];
+  color: string;
+  compositeOperation: GlobalCompositeOperation;
```

Comment:
penColor is already part of PointGroupOptions. compositeOperation should be part of PointGroupOptions.

Reply:
You're right! This is completely obsolet.
I just done copy + pasted without thinking 😄